Neos.Neos/Classes/Domain/Service/FusionService.php

```diff
@@ -257,15 +243,14 @@ protected function getFusionIncludes(array $fusionResources)
     protected function prepareAutoIncludeFusion(): array
     {
         $autoIncludeFusion = [];
-        foreach (array_keys($this->packageManager->getAvailablePackages()) as $packageKey) {
+        foreach ($this->packageManager->getAvailablePackages() as $packageKey => $package) {
             if (isset($this->autoIncludeConfiguration[$packageKey]) && $this->autoIncludeConfiguration[$packageKey] === true) {
                 $autoIncludeFusionFile = sprintf($this->autoIncludeFusionPattern, $packageKey);
                 if (is_file($autoIncludeFusionFile)) {
                     $autoIncludeFusion[] = $autoIncludeFusionFile;
                 }
-                $autoIncludeNodeTypesFolder = sprintf($this->autoIncludeNodeTypesFolderPattern, $packageKey);
-                if (is_dir($autoIncludeNodeTypesFolder)) {
-                    $autoIncludeFusion[] = $autoIncludeNodeTypesFolder . '/**/*.fusion';
+                if (is_dir($package->getPackagePath() . '/NodeTypes')) {
```

Comment:
i dont really like the mixture of `getPackagePath()` and `package://` side by side in this FusionService

i think you did this for performance reasons?

but either way -> can later be refactored - or not :P